packages/engine-rn/templates/platforms/ios/RNVApp/AppDelegate.mm

```diff
@@ -5,7 +5,27 @@
 
 @implementation AppDelegate
 
-{{APPDELEGATE_METHODS}}
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
```

Comment:
this should not be touched, if there are methods that need to be changed then they should be changed in APPDELEGATE_METHODS injection